packages/engine-core/src/binary/BinaryEngine.ts

```diff
@@ -519,7 +519,8 @@ ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
       }
       if (this.engineEndpoint) {
         try {
-          await pRetry(() => this.connection.get('/'), {
+          this.connection.open(this.engineEndpoint)
+          await pRetry(() => this.connection.get('/status'), {
```

Comment:
Unrelated: Do we do this waiting for status to work in the normal BinaryEngine flow as well?

Reply:
No, normally on startup we check logs to know the binary engine has started.

Reply:
Ah right, because we trigger the start and then can watch the logs, that is easier and equally reliable.